pkg/agent/nodeportlocal/portcache/port_table.go

```diff
@@ -25,6 +25,11 @@ import (
 	"antrea.io/antrea/pkg/agent/nodeportlocal/rules"
 )
 
+const (
```

Comment:
Its not only because of these predefined constants that these tests are failing but also there are many functions, so if we use the copy approach then we will have to copy all the functions also from these files to port_table.go